packages/lib/services/action.ts

```diff
@@ -0,0 +1,66 @@
+import { prisma } from "@formbricks/database";
+import { DatabaseError } from "@formbricks/errors";
+import { EventType } from "@prisma/client";
+
+export type TranformEventClassOutput = {
+  id: string;
+  createdAt: Date;
+  updatedAt: Date;
+  name: string;
+  description: string | null;
+  type: EventType;
+  noCodeConfig: any;
+  environmentId: string;
+  eventCount: number;
```

Comment:
please rename `eventCount` to `actionCount` 😊
We will now try to be consistent and replace `eventClass` with `actionClass` and `event` with `action` 😊